MREUnityRuntime/MREUnityRuntimeLib/Core/Actor.cs

```diff
@@ -79,12 +79,13 @@ public new string Name
         internal MWTransform LocalTransform => transform.ToMWTransform();
 
         internal Guid MaterialId { get; set; } = Guid.Empty;
-
+        
         internal bool Grabbable { get; private set; }
         
-        private bool AppearanceEnabled { get; set; } = true;
-
-        private bool ActiveAndEnabled => ((Parent as Actor)?.ActiveAndEnabled ?? true) && AppearanceEnabled;
+        private UInt32 AppearanceEnabled = UInt32.MaxValue;
```

Comment:
cslint: private members should start with a lowercase letter : `AppearanceEnabled` -> `appearanceEnabled`

Reply:
Weird, I would have sworn I got just the opposite comment in a different PR, but I can't find it now. Changing it.

---
In reply to: [267969292](https://github.com/Microsoft/mixed-reality-extension-unity/pull/60#discussion_r267969292) [](ancestors = 267969292)